utils/createToken.js

```diff
@@ -1,9 +1,12 @@
 const jwt = require('jsonwebtoken');
```

Comment:
Please, rename this file to the `create-token` to follow a common naming patterns

Reply:
renamed